jetty/install_windows_src.md

```diff
@@ -142,22 +119,41 @@ If there are no errors, all the binaries should be ready to use.
 ## Using the workspace
 
 The workspace needs to be sourced every time a new terminal is used (
-and Conda environment activated before that).
+and the Pixi environment activated before that).
 
 The overall instructions for setting up a new terminal to use the built
 workspace are:
 
 ```bash
 # CMD
-conda activate gz-ws
+cd gazebo
+pixi shell
 call install\setup.bat
 
 # PowerShell
-conda activate gz-ws
+cd gazebo
+pixi shell
 .\install\setup.ps1
 ```
+<div class="warning">
+An issue in the conda-forge Qt6 package is requiring to set QT environment variables:
+</div>
+
+```bash
+# Until https://github.com/conda-forge/qt-main-feedstock/issues/275 is resolved
+cd gazebo
+set QT_PLUGIN_PATH=%CD%\.pixi\envs\default\Library\lib\qt6\plugins
```

Comment:
Actually, in the pixi shell the CONDA_PREFIX environment variable is active as I've learned!

```suggestion
set QT_PLUGIN_PATH=%CONDA_PREFIX%\Library\lib\qt6\plugins\platforms
```